packages/test/test-end-to-end-tests/src/test/gc/gcUnreferencedFlagInSnapshot.spec.ts

```diff
@@ -64,41 +49,41 @@ describeFullCompat("GC unreferenced flag validation in snapshot", (getTestObject
         runtimeOptions,
     );
 
+    const logger = new TelemetryNullLogger();
     let mainContainer: Container;
     let mainDataStore: TestDataObject;
     let documentStorage: IDocumentStorageService;
-    let summaryCollection: SummaryCollection;
 
     let latestUploadedSummary: ISummaryTree | undefined;
     let latestSummaryContext: ISummaryContext | undefined;
+    // Stores the latest acked summary for the document.
+    let latestAckedSummary: IAckedSummary | undefined;
 
     const createContainer = async (): Promise<Container> => {
         return await provider.createContainer(runtimeFactory) as Container;
     };
 
-    /**
-     * Waits for a summary with the current state of the document (including all in-flight changes). It basically
-     * synchronizes all containers and waits for a summary that contains the last processed sequence number.
-     * @returns the version of this summary that be used to download it from the server.
-     */
-     async function waitForSummary() {
-        await provider.ensureSynchronized();
-        const sequenceNumber = mainContainer.deltaManager.lastSequenceNumber;
-        const ackedSummary: IAckedSummary =
-            await summaryCollection.waitSummaryAck(sequenceNumber);
-        return {
-            summaryVersion: ackedSummary.summaryAck.contents.handle,
-            sequenceNumber,
-        };
-    }
+    const getNewSummarizer = async () => {
+        return loadSummarizer(provider, runtimeFactory, mainContainer.deltaManager.lastSequenceNumber);
+    };
 
     /**
      * Callback that will be called by the document storage service whenever a summary is uploaded by the client.
+     * Update the summary context to include the summary proposal and ack handle as per the latest ack for the
+     * document.
      */
-    function uploadSummaryCb(summaryTree: ISummaryTree, context: ISummaryContext): ISummaryContext {
+     function uploadSummaryCb(summaryTree: ISummaryTree, context: ISummaryContext): ISummaryContext {
```

Comment:
Nit: an extra space seems to have crept in at the start of the line.

Reply:
Fixed it, thanks!